muikku-core-plugins/src/main/resources/META-INF/resources/jsf/workspace/structure.txt

```diff
@@ -0,0 +1,41 @@
+FIRST I GOT TO SETUP THE MATERIALS CONFIGURATION UPDATE (that will be hard)
```

Comment:
Is this file relevant to the functionality of the Thing?

Reply:
I would think not. Deleted it.